popup/popup.js

```diff
@@ -26,11 +28,16 @@ function listenForClicks() {
                     return;
                 case "Reset":
                     // TODO: implement
+                    const tabs = await browser.tabs.query({
+                        currentWindow: true,
+                        active: true
+                    }).catch(onError);
+                    sendMessageToTabs(tabs, { "reset": "true" });
+                    alert("Please refresh your browser tab to apply.");

```

Comment:
Force refresh after resetting?

Reply:
So I've actually tried to reload a fresh (default) localStorage save in real time, right after clicking reset, but for some reason it crashes the whole add-on.
P.S 
the whole thing is implemented on the `storageLoader.js` file.

Reply:
After resetting the localStorage have you tried using location.reload()?

Reply:
Oh you want us to refresh the page for him? but If the user have something important that he didn't save (e.g. a forum post)
it will delete it.

Reply:
Yeah, you have got a point, so can we revert the page to the default?
Like removing the CSS file from the doc?

Reply:
Yes you can but as I said, I tried to implement it but failed.
I will try again though.
(for now if you refresh the page it will go back to normal)